department-of-veterans-affairs/caseflow
# Create better hearings data for demo

Relates to https://vajira.max.gov/browse/CASEFLOW-990

### Description
Based on discussion with Product and Design on this [thread](https://dsva.slack.com/archives/C01155MJGF2/p1614632661063300?thread_ts=1614272047.034900&cid=C01155MJGF2), this PR refactors existing hearings seed script to create better data, specifically focusing on varied hearing times, varied judges for hearing days, varied poas, and varied Veteran/Claimant data. 
This also creates varied request issues for unscheduled hearings.
Additionally this PR adds user `BVASYELLOW` to hearings admin.
#### Testing Plan
1. run `make reset`
2. login as `BVASYELLOW` and go to http://localhost:3000/hearings/schedule
3. Poke around to see that the data is varied
